Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/RequestPasswordViewController.swift

```diff
@@ -49,12 +44,12 @@ class RequestPasswordViewController: UIAlertController {
         let okAction = UIAlertAction(title: okTitle, style: .default) { [unowned controller] (action: UIAlertAction) -> Void in
             if let passwordField = controller.textFields?[0] {
                 let password = passwordField.text ?? ""
-                controller.callback?(Either.left(password))
+                controller.callback?(Result<String>.success(password))
```

Comment:
This should do as well:
```suggestion
                controller.callback?(.success(password))
```